gg_interface/src/main.cpp

```diff
@@ -1,34 +1,32 @@
-#include <Arduino.h>
-#include <ArduinoJson.h>
-#include <Adafruit_INA260.h>
-
 #include "main.h"
 
 Adafruit_INA260 ina260 = Adafruit_INA260();
+bool initialized = false;
 volatile bool ledState = true;
 
 void setup() {
     // Every embedded system needs an LED
-    pinMode(ledPin, OUTPUT);
-    digitalWrite(ledPin, HIGH);
+    pinMode(LED_PIN, OUTPUT);
+    digitalWrite(LED_PIN, HIGH);
 
     // Configure serial port
-    Serial.begin(115200);
+    Serial.begin(SERIAL_BAUD);
 
     // Wait until serial port is opened
-    while (!Serial) { delay(10); }
-    Serial.println("Gufu Gud");// INA260 Test");
+    while (!Serial) {
+        delay(10);
+    }
+    Serial.println("Gufu Gud");
 
     // Initilize the INA260
     if (!ina260.begin()) {
-        Serial.println("Couldn't find INA260 chip");
-        while (1);
+        Serial.println("Couldn't find INA260!");
+        return;
```

Comment:
Should we do any retries here instead of hard-failing after one attempt?

Reply:
Good call!